exchanges/gateio/gateio_websocket.go

```diff
@@ -721,22 +721,22 @@ func (g *Gateio) handleSubscription(event string, channelsToSubscribe []stream.C
 	if err != nil {
 		return err
 	}
-	var errs common.Errors
+	var errs error
 	for k := range payloads {
 		result, err := g.Websocket.Conn.SendMessageReturnResponse(payloads[k].ID, payloads[k])
 		if err != nil {
-			errs = common.AppendError(errs, err)
+			_ = common.AppendError(errs, err)
```

Comment:
need to assign, blanking out does nothing. And below.

Reply:
even if I don't assign a value to it, my error field (errs) gets updated.
Plus, linter raises 'ineffectual assignment to variable errs' because I don't use the assigned value immediately

Reply:
```go
func TestXxx(t *testing.T) {
	var errs error

	_ = common.AppendError(errs, errors.New("this is an error"))
	_ = common.AppendError(errs, errors.New("this is another error"))
	_ = common.AppendError(errs, errors.New("this is the last error"))

	fmt.Println("errs", errs)
}
```
```
=== RUN   TestXxx
errs <nil>
```

Use nolint directive